cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -49,7 +50,7 @@ public class KafkaConfiguration extends AbstractConfiguration {
     private static final List<String> FORBIDDEN_PREFIX_EXCEPTIONS;
 
     static {
-        FORBIDDEN_PREFIXES = AbstractConfiguration.splitPrefixesToList(KafkaClusterSpec.FORBIDDEN_PREFIXES);
+        FORBIDDEN_PREFIXES = AbstractConfiguration.splitPrefixesToList(String.join(", ", KafkaClusterSpec.FORBIDDEN_PREFIXES, QuotasPluginStrimzi.FORBIDDEN_OPTIONS));
```

Comment:
I wasn't sure if I should add those to the `FORBIDDEN_PREFIXES`, as those are not exactly prefixes + it's a quite long list. But if it's okay, I will definitely add them to the `KafkaClusterSpec`.

Reply:
What does it mean `those are not exactly prefixes`? If they are not exactly prefixes, they should not be in forbidden prefixes I guess?

Reply:
For the Kafka Static Quotas plugin, there are exact config options that we are configuring and they are forbidden. But those are not prefixes, as user can specify for example `client.quota.callback.static.storage.check-interval`. I had there method for ignoring those config options previously. So maybe instead of adding the options to the `FORBIDDEN_PREFIXES`, I can have it as part of the `validateConfigurationOfQuotasPlugin` (similarly to `validateConfiguration`)?

Reply:
The question is what do you want to achieve. But I think the forbidden prefixes are fine -> there are already some complete options as well in there already I think.

Reply:
Okay, yeah I didn't know if it's fine to add them all there. Fixed, thanks.